module/userExtension/helper.js

```diff
@@ -0,0 +1,242 @@
+
+let sunBirdService =
+    require(ROOT_PATH + "/generics/services/sunbird");
+
+/**
+* user related information be here.
+* @method
+* @class  userExtensionHelper
+*/
+module.exports = class userExtensionHelper {
+
+
+    /**
+    * getProfile.
+    * @method
+    * @name  getProfile
+    * @param  {queryObject, token}  - requested queryObject and token.
```

Comment:
@rakeshSgr , @deepakr123 - Let all params be in a new line, each with its own explanation